components/ProcessSteps/src/SubStepHeading.tsx

```diff
@@ -1,6 +1,6 @@
 import React, { PropsWithChildren } from 'react';
 
-export const SubStepHeading = ({ children }: PropsWithChildren<{}>) => (
+export const SubStepHeading = ({ children }: PropsWithChildren<NonNullable<unknown>>) => (
```

Comment:
Moet het complexer, of kan het juist simpeler?

```suggestion
export const SubStepHeading = ({ children }: PropsWithChildren) => (
```